packages/eslint-config/svelte.cjs

```diff
@@ -39,5 +39,30 @@ module.exports = {
         'no-undef-init': 'off',
       },
     },
+    // Rules for tests
+    {
+      files: ['**/__tests__/**', '**/*.test.ts', '**/*.spec.ts'],
```

Comment:
Same comment as last PR: Since we are enforcing tests with the `.spec.ts` extension, can we drop `**/*.test.ts`?

Also, is there a good way to enforce that test files are in an `__tests__` dir?